jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MetaData.java

```diff
@@ -331,4 +331,121 @@ public String toString()
             return String.format("%s{s=%d,h=%d,cl=%d}", getHttpVersion(), getStatus(), headers.size(), getContentLength());
         }
     }
+
+    /**
+     * <p>Marker interface to signal that a {@link MetaData} could not be created,
+     * for example due to an invalid URI or invalid headers.</p>
+     */
+    public interface Failed
```

Comment:
I know it is not needed, but the methods are MetaData only, so the interface should extend
```suggestion
    public interface Failed extends MetaData
```

Reply:
@gregw `MetaData` is a class, so I cannot do the above.